nistats/second_level_model.py

```diff
@@ -422,3 +423,148 @@ def compute_contrast(
         output.get_header()['descrip'] = (
             '%s of contrast %s' % (output_type, contrast_name))
         return output
+
+    def compute_contrast_permutations(
+            self, second_level_contrast='intercept', first_level_contrast=None,
+            stat_type=None, output_type='cor_z_score', threshold=0.001,
+            height_control='fpr', two_sided_test=True, n_perm=10000,
+            random_state=None):
+        """Generate different permutation test outputs corresponding to
+        the contrasts provided e.g. corrected p values and cluster statistics.
+
+        Parameters
+        ----------
+        second_level_contrast: str or array of shape (n_col), optional
+            Where ``n_col`` is the number of columns of the design matrix,
+            The string can be a formula compatible with the linear constraint
+            of the Patsy library. Basically one can use the name of the
+            conditions as they appear in the design matrix of
+            the fitted model combined with operators /*+- and numbers.
+            Please checks the patsy documentation for formula examples:
+            http://patsy.readthedocs.io/en/latest/API-reference.html#patsy.DesignInfo.linear_constraint
+            By default it returns the group contrast of the main effect.
+
+        first_level_contrast: str or array of shape (n_col) with respect to
+                              FirstLevelModel, optional
+            In case a list of FirstLevelModel was provided as
+            second_level_input, we have to provide a contrast to apply to
+            the first level models to get the corresponding list of images
+            desired, that would be tested at the second level.
+
+        stat_type: {'t', 'F'}, optional
+            Type of the contrast
+
+        output_type: str, optional
+            Type of the output map. Can be 'cor_p_value', 'cor_z_score',
```

Comment:
I am just providing the p_values or their transformation into z scores, the same for corrected p-values, but considering its just the application of one function and confusing in the interface, should I take this out? 
I Included it as a way to promote using z scores instead of the T of F stats.